dev-board/Src/imu-driver/bno055.cpp

```diff
@@ -0,0 +1,133 @@
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdarg.h>
+
+#include "bno055.h"
+#include "main.h"
+
+I2C_HandleTypeDef hi2c1;
+static uint8_t bno055ReadReg(uint8_t reg);
+static void bno055SendReg(uint8_t reg, uint8_t data);
+static int16_t bno055ReadData(uint8_t lsbReg);
+static struct bno055_3axis bno055Read3Axis(uint8_t reg);
+static void MX_I2C1_Init(void);
+
+
+void bno055Init()
```

Comment:
Can you split up your code into different sections like https://github.com/Solar-Gators/Telemetry_2019-2020/blob/master/templates/c-header-template.h#L8 for stuff like include files, defines, structs, public declartions etc